android/app/src/main/java/com/google/firebase/samples/apps/friendlypix/PostViewHolder.java

```diff
@@ -141,13 +142,18 @@ public void setPostClickListener(PostClickListener listener) {
     }
 
     public void setLikeStatus(LikeStatus status, Context context) {
-        mLikeIcon.setImageDrawable(ContextCompat.getDrawable(context,
-                status == LikeStatus.LIKED ? R.drawable.heart_full : R.drawable.heart_empty));
+//        Check if we still have a reference to the ImageView
+        if (mLikeIcon == null && mView != null)
```

Comment:
Please add braces surrounding the 'if' block body according to the style guide: https://source.android.com/source/code-style.html#use-standard-brace-style